api/community/serializers.py

```diff
@@ -22,6 +25,9 @@ class Meta:
         fields = '__all__'
 
 class MediaSerializer(serializers.ModelSerializer):
+    hashtag = serializers.CharField()
+    group = serializers.CharField()
+    topics = serializers.HyperlinkRelatedField(many=True, )
```

Comment:
This should be [HyperlinkedRelatedField](https://www.django-rest-framework.org/api-guide/relations/#hyperlinkedrelatedfield)

Reply:
This is throwing errors so until this is fixed I can't test out the fixtures

Reply:
removed serializer definition for topics for the time being